src/main/java/build/buildfarm/instance/shard/RedisShardBackplane.java

```diff
@@ -1390,7 +1396,10 @@ public boolean canPrequeue() throws IOException {
   @Override
   public BackplaneStatus backplaneStatus() throws IOException {
     BackplaneStatus.Builder builder = BackplaneStatus.newBuilder();
-    builder.addAllActiveWorkers(client.call(jedis -> state.executeAndStorageWorkers.keys(jedis)));
+    builder.addAllActiveWorkers(
+        client.call(
+            jedis ->
+                Sets.union(state.executeWorkers.keys(jedis), state.storageWorkers.keys(jedis))));
```

Comment:
Side effect of this change:
Prometheus started reporting incorrect count of workers for us.
Reason: We register workers with execution and cache capabilities both but we only utilize cas capabillities. While `storageWorkers` map is update on every `fetchAndExpireStorageWorker` call but `deregisterWorker` method is not called from `stop` method (as per my understanding, I may be wrong) and workers stored in `executeWorkers` maps remains in redis forever.